features/lib/osrm_loader.js

```diff
@@ -45,11 +45,11 @@ class OSRMBaseLoader{
         var retryCount = 0;
         let retry = (err) => {
           if (err) {
-            if (retryCount < 10) {
+            if (retryCount < this.scope.OSRM_CONNECTION_RETRIES) {
```

Comment:
We could also add some exponential back-off, which might be more robust to fluctuations in CI startup times.

Reply:
Added `OSRM_CONNECTION_EXP_BACKOFF_COEF`, please take a look.